requirements.txt

```diff
@@ -78,7 +76,7 @@ docopt==0.6.2
     #   notifications-python-client
 docutils==0.15.2
     # via awscli
-eventlet==0.30.2
+eventlet==0.30.2 # pyup: ignore
```

Comment:
Should we keep this?

Reply:
0aa7cf1aaff9dd8fd4ec1fb09fa6d64226e159b1

> **Tell Pyup to ignore outdated Eventlet version**
>
> We already do this in the admin app:
> https://github.com/alphagov/notifications-admin/pull/3876/files
>
> Upgrading Eventlet is blocked until this change in Gunicorn is released:
> https://github.com/benoitc/gunicorn/pull/2581/files

Reply:
I’m not sure why Pyup wasn’t picking this up before, but it is now

Reply:
(Sorry, I read the diff the wrong way round 🙈.)